tronbyt_server/__init__.py

```diff
@@ -275,11 +275,13 @@ def create_app(test_config: Optional[Dict[str, Any]] = None) -> Flask:
     def timeago(seconds: int) -> str:
         if seconds == 0:
             return str(_("Never"))
-        return format_timedelta(
-            dt.timedelta(seconds=seconds - int(time.time())),
-            granularity="second",
-            add_direction=True,
-            locale=get_locale(),
+        return str(
```

Comment:
Why was this needed? format_timedelta already returns `str`

Reply:
evidently mypy didn't realize that.